protocol/avdl/keybase1/metadata.avdl

```diff
@@ -41,10 +41,15 @@ protocol metadata {
          KeyBundle ReaderBundle;
   }
 
+  record LockContext {
+    long    requireLockID;
```

Comment:
@jzila suggested using a 64-bit integer in database, since that's enough space to avoid collision within the same TLF space. If there is collision, we'd just have reduced concurrency due to locking, but it won't affect correctness.

Would you prefer a `[8]byte`? Although there's no way to enforce that in avdl and it'd be just `[]byte`, which is a bit weird compared to a fixed size int I think.